app/backplane/sensor_module/src/c_sensing_tenant.cpp

```diff
@@ -60,7 +78,7 @@ void CSensingTenant::Run() {
         data.ImuGyroscope.Y = imuGyroscope.GetSensorValueFloat(SENSOR_CHAN_GYRO_Y);
         data.ImuGyroscope.Z = imuGyroscope.GetSensorValueFloat(SENSOR_CHAN_GYRO_Z);
 
-#ifndef CONFIG_ARCH_POSIX // TODO: No magnetometer simulation capability yet
```

Comment:
like the suns return 0 so do you still need the ifndef here at all? if your remove the comment did you mean to remove the whole yhing

Reply:
> suns return 0
wut?

Nah did not intend to remove the whole thing or intend to remove the comment acutally lol

Reply:
"sims" not "suns" lol not my most coherent comment. Thats what i get for code reviewing on the github app. but ok if you didnt intend to remove the comment either that makes more sense. i was thinking you should i either remove the ifdefs and comment or neither but was confused why you would remove one. 

afaik, though, the skeleton for magnetometer sims is on main. is there any reason to keep the ifdef rather than just getting the 0 from the magnetometer sim?

Reply:
We dont have a magnetometer simulation driver is the thing though. Brought back the TODO that was accidentally deleted

Reply:
ah so like morally that is correct we don't have mag sims. 

but the mag openrocket sensor is stubbed out enough to return 0s

I get it if you wanna leave the ifsefs as a reminder that that data isn't valid but you can get a compileable output using sim magnetometer if you want